aws/components/secretstore/id.ftl

```diff
@@ -0,0 +1,86 @@
+[#ftl]
+
+[@addResourceGroupInformation
+    type=SECRETSTORE_COMPONENT_TYPE
+    attributes=[
+        {
+            "Names": "Engine",
+            "Values" : [ "secretsmanager" ],
+            "Default" : "secretsmanager"
```

Comment:
As the `shared` provider doesn't grant the value "secretsmanager" I believe this new value will resolve in-engine to `aws:secretsmanager`. 

The default values aren't namespaced by the engine, so that would mean that the default value must be namespaced manually.

We could update the engine to do that for us but it would prevent a provider defaulting to a value that was initially defined by the shared provider so seems best to leave as it is. Hard to document that though...